### PR TITLE
Remove T4 nodepool for RAG

### PR DESCRIPTION
**Pull request by imreddy13**
- Switch Ray workers to L4s

### CODE REVIEW

File: modules/kuberay-cluster/main.tf [hunk 1]
```diff
@@ -68,7 +68,7 @@ resource "helm_release" "ray-cluster" {
         "iam.gke.io/gke-metadata-server-enabled" : "true"
         } : var.enable_gpu ? {
         "iam.gke.io/gke-metadata-server-enabled" : "true"
-        "cloud.google.com/gke-accelerator" : "nvidia-tesla-t4"
+        "cloud.google.com/gke-accelerator" : "nvidia-l4"
```
Comment (andrewsykim) [inline modules/kuberay-cluster/main.tf:71]: We also discussed just running the Ray cluster on CPUs right? 

Have we tested if generating the embeddings with CPUs is fast enough?

Comment (imreddy13) [reply modules/kuberay-cluster/main.tf:71]: @blackzlq played with this but we haven't benchmarked, next PR

**Review by bjornsen — APPROVED** [review]

**Comment by imreddy13** [comment]
AP test is broken. standard test passed on a previous run (only change is TF linting). bypassing

**Pull request merged**